pydantic_ai_slim/pydantic_ai/models/mistral.py

```diff
@@ -110,7 +110,7 @@ class MistralModel(Model):
     json_mode_schema_prompt: str = """Answer in JSON Object, respect the format:\n```\n{schema}\n```\n"""
 
     _model_name: MistralModelName = field(repr=False)
-    _system: str | None = field(default='mistral', repr=False)
+    _system: str = field(default='mistral_ai', repr=False)
```

Comment:
is this change just your opinion or a documented convention or whatever? (I'm okay either way just want to understand)